![alle_commits](https://github.com/TordWestby/ITF20319_Oblig3/assets/102856887/2d9286db-2598-4276-9586-8b3cc3058b53)# ITF20319_Oblig3
#Oblig 3 for Software Engineering

Jeg startet oppgaven med å opprette et nytt repository, jeg lastet da ned oblig 2 og la til filene jeg hadde levert der. 
Etter det begynte jeg å lese på dokumentasjonen til [Github Actions](https://docs.github.com/en/actions/learn-github-actions) og [Building and testing Python](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python)
Jeg kjørte så en commit hvor jeg med vilje faila testen min slik at jeg kan vise hvordan det ser ut og det ble da seende slik ut: ![Skjermbilde av en commit på github hvor pytesten faila.](![Skjermbilde 2023-10-25 144517](https://github.com/TordWestby/ITF20319_Oblig3/assets/102856887/9ed61837-2f1b-4c52-8132-bb91445f3418)
)

Etter dette endret jeg på koden slik at jeg vil få en suksesfull test så jeg gjorde endringene jeg måtte i koden også sjekket jeg 

[Contribution guidelines for this project](docs/CONTRIBUTING.md)
